docker/dev/build.sh

```diff
@@ -71,7 +71,7 @@ if [[ ! -d models ]]; then
     git clone https://github.com/sql-machine-learning/models
 fi
 cd models
-git checkout c897963f821d515651de79cb4ef1fbf6126ecaa5
+git checkout 07a75db
```

Comment:
@shendiaomo Is there a reason that we need this special version of `models`? Should we comment and document the reason, or should we git tag the models repo?